python_modules/libraries/dagster-snowflake/setup.py

```diff
@@ -32,6 +32,6 @@ def get_version() -> str:
         ],
         packages=find_packages(exclude=["dagster_snowflake_tests*"]),
         install_requires=[f"dagster{pin}", "snowflake-connector-python>=2.1.0"],
-        extras_require={"snowflake.sqlalchemy": ["sqlalchemy", "snowflake-sqlalchemy"]},
+        extras_require={"snowflake.sqlalchemy": ["sqlalchemy", "snowflake-sqlalchemy", "pandas"]},
```

Comment:
The effect that this will have is that, if someone does `pip install dagster[snowflake.sqlalchemy]`, then sqlalchemy, snowflake-sqlalchemy, and pandas will all be installed. However, I think we want to allow people to depend on sqlalchemy without depending on pandas.  If they want both, they could do `pip install dagster[snowflake.sqlalchemy,pandas]`. I believe this requires adding a "pandas" key to the dictionary instead of adding a value to the list for the existing key.

Reply:
Thanks!
I'm fairly new with setuptools, should it be:
```
extras_require={
    "snowflake.sqlalchemy": ["sqlalchemy", "snowflake-sqlalchemy"],
    "snowflake.pandas": ["pandas"]}
```
or should the second dict item be just "pandas" vs snowflake.pandas?

Reply:
I think the second dict item should just be "pandas".

Reply:
Updated per @sryza  suggestions. Thanks!